### PR TITLE
DOP-5022: Adding netlify integration for mut-publish

### PR DESCRIPTION
**Pull request by biancalaube**
### Stories/Links

[DOP-5022](https://jira.mongodb.org/browse/DOP-5022)
[git-changed-files-testing](https://app.netlify.com/sites/git-changed-files-testing-bianca/deploys)
[bianca-fork-docs-landing](https://app.netlify.com/sites/bianca-fork-docs-landing/deploys)

### Notes / Blockers

* Everything currently works for the test case of enabling this integration on 'bianca-fork-docs-landing" in the s3 buck of **docs-mongodb-org-dotcomstg** under the prefix **/netlify/docs-qa**
    * While Brandon and I were pair programming we discussed that it was safe to use to the docs-mongodb-org-dotcomstg s3 bucket with that prefix since its the staging prod under a new prefix so it wouldn't mess up anything in prod
* I've added some comments throughout the code about stuff that maybe needs changing that I'm unsure of. Such as how many logs/comments I should be making throughout.
* In my `mut-publish` command I'm unsure of which bucket we would want to use in prod/what prefix should be used. I added more context in the comments of the code for this question and what should maybe be changed.
*  I know this is eventually going to be its own integration but right now I have added the integration to git-changed-files integration. However, I made a copy so the integration should be under **git-changed-files-testing** in netlify

### Testing
* Enable the git-changed-files-testing integration on a forked content repo
* deploy the content repo
* you should see the content in the linked s3 bucket

### CODE REVIEW

File: git-changed-files/src/searchConnector.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+export const getSearchDb = async () => {
+	console.log("getting search db");
+	const uri = ATLAS_SEARCH_URI;
+	const dbName = SEARCH_DB_NAME;
+	const searchDbClient = await db({ uri, dbName });
+	return searchDbClient;
+};
+
+export const getSnootyDb = async () => {
```
Comment (branberry) [inline git-changed-files/src/searchConnector.ts:41]: @anabellabuckvar has some great code in her PR that has these refactored in a way that works well, especially with tests. 

Don't worry about making those changes now, though. I think our next step before we go much further is to create a shared project.

**Review by branberry — COMMENTED** [review]
Hey Bianca, overall this looks great! Just a few nits.

File: git-changed-files/src/getProperties.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    DocsetsDocument,
+    ReposBranchesDocument,
+} from "./types";
+import {
+    getCollection,
+    getSnootyDb,
+} from "./searchConnector";
+import type { Collection } from 'mongodb';
+
+export const getProperties = async (repo_name: string) => {
```
[the remark this answers is not in the document]
Comment (biancalaube) [reply git-changed-files/src/getProperties.ts]: fixed!

File: git-changed-files/src/getProperties.ts [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    //connect to database and get repos_branches, docsets collections
+    console.log("connectiong to mongodb...");
+    const dbSession = await getSnootyDb();
+    const repos_branches = getCollection(dbSession, "repos_branches");
```
[the remark this answers is not in the document]
Comment (biancalaube) [reply git-changed-files/src/getProperties.ts]: thanks for the feedback, fixed !

**Pull request closed without merging**